python/pyspark/sql/functions/builtin.py

```diff
@@ -17723,7 +17723,7 @@ def array_sort(
 
 
 @_try_remote_functions
-def shuffle(col: "ColumnOrName") -> Column:
+def shuffle(col: "ColumnOrName", seed: Optional[Union[Column, int]] = None) -> Column:
```

Comment:
just curious, why do we want to support column type here?

Reply:
It's actually written on the top. e.g., in case we support `seed` as non-foldable expression in the future.